scene/gui/separator.cpp

```diff
@@ -32,7 +32,7 @@
 Size2 Separator::get_minimum_size() const {
 
 	Size2 ms(3, 3);
-	ms[orientation] = get_constant("separation");
+	ms[1 - orientation] = get_constant("separation");
```

Comment:
It took me some thinking to understand why it's needed, and especially I had to check how `Orientation` is defined in `core/math/math_2d.h`. I think we'd win by being a bit more explicit here, e.g. with:
```cpp
if (orientation == VERTICAL) {
	ms.x = get_constant("separation");
else { // HORIZONTAL
	ms.y = get_constant("separation");
}
```

Reply:
i totally agree